rules/src/main/resources/io/github/belgif/rest/guide/validator/rules/oas/Rule-VerifyOperationId.drl

```diff
@@ -0,0 +1,87 @@
+package io.github.belgif.rest.guide.validator.rules.oas;
+
+import io.github.belgif.rest.guide.validator.core.ApiFunctions
+import io.github.belgif.rest.guide.validator.core.parser.Parser.ParserResult;
+global io.github.belgif.rest.guide.validator.core.parser.Parser.ParserResult parserResult;
+import io.github.belgif.rest.guide.validator.core.ViolationReport;
+global io.github.belgif.rest.guide.validator.core.ViolationReport oas;
+import java.util.Set;
+import java.util.List;
+import java.util.Map;
+import io.github.belgif.rest.guide.validator.core.model.OperationDefinition;
+import org.eclipse.microprofile.openapi.models.Operation
+import java.util.HashMap
+import java.util.HashSet
+import java.util.ArrayList;
+
+function String printAllJsonPointers(List<OperationDefinition> operations) {
```

Comment:
you can use the join method to shorten this: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/String.html#join(java.lang.CharSequence,java.lang.Iterable)